templates/repo/branch/list.tmpl

```diff
@@ -27,22 +27,36 @@
 							</td>
 							<td class="right aligned overflow-visible">
 								{{if and $.IsWriter (not $.Repository.IsArchived) (not .IsDeleted)}}
-									<button class="ui basic jump button icon show-create-branch-modal" data-tooltip-content="{{$.locale.Tr "repo.branch.new_branch_from" ($.DefaultBranch)}}" data-branch-from="{{$.DefaultBranch}}" data-branch-from-urlcomponent="{{PathEscapeSegments $.DefaultBranch}}" data-modal="#create-branch-modal">
+									<button class="ui tertiary button show-modal show-create-branch-modal gt-mx-3"
```

Comment:
```suggestion
									<button class="ui tertiary button show-create-branch-modal gt-mx-3"
```

I think we need to remove this `show-modal`, to make sure the code is the same as commit view page (and the same as old code)